lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1789,11 +1804,23 @@ def get_proxy_port_no(self) -> Optional[int]:
     # RYU rest API is not able dump flows from non zero table.
     # this adds similar API using `ovs-ofctl` cmd
     def get_flows(self, table_id) -> []:
+        """Fetch the OVS flow rules"""
         output = self.magma_utils.exec_command_output(
-            "sudo ovs-ofctl dump-flows gtp_br0 table={}".format(table_id),
+            "sudo ovs-ofctl dump-flows gtp_br0 table={0}".format(table_id),
         )
         return output.split("\n")
 
+    def delete_flows(self, table_id):
```

Comment:
What is the difference between this function and delete_ovs_flow_rules?

Reply:
delete_ovs_flow_rules function is used to expose the delete_flows function under gtp bridge utility to s1ap utillity class similar to other functions. All the raw ovs flow rules operations are wrapped under functions in gtp bridge utility class.